packages/shared/locales/en.json

```diff
@@ -762,6 +762,11 @@
             "bodyMerge": "Your funds will be staked for {airdrop}. To change which airdrops you want to participate in, unstake this wallet and stake it again.",
             "estimatedAirdrop": "Estimated rewards"
         },
+        "newStakingPeriodNotification": {
+            "title": "Staking round 2 from March 22nd",
+            "body": "Ready for the next round of staking? Participate in the 30 days staking round 2 from March 22nd. Stake your IOTAs and earn ASMB rewards!",
+            "info": "Shimmer staking is now complete and only ASMB tokens will be distributed."
+        },
         "shimmer-info": {
```

Comment:
This isn't a part of your changes, but shouldn't this be `shimmerInfo` for consistency?

Reply:
That and the assembly-info don't even appear to be used in the application on develop branch?